### ISSUE TITLE
Add `role` to `<iconify-icon>` tag for accessibility

### ISSUE THREAD

**Issue by cwickham** [mcanouil/quarto-iconify#28]
I'm getting the following AxDevTools Accessibility issue when using an icon on my website:
> `aria-label` attribute cannot be used on a `iconify-icon` with no valid `role` attribute.

<img width="754" alt="Screenshot 2024-07-16 at 3 43 40 PM" src="https://github.com/user-attachments/assets/98c6868c-7155-4135-8dfa-6f668f22e04a">

I'm no expert, but perhaps adding `role="img"` to the tag might solve it.

**Comment by mcanouil** [comment]
Thanks, I'll look into it.
There is probably something in Iconify for this.

**Comment by mcanouil** [comment]
Role is indeed not added by Iconify and needs to be added (even if it does not solve the issue): https://iconify.design/docs/icon-components/svg-framework/render-icon.html#result

**Comment by mcanouil** [comment]
Adding role did fix the issue and no other issue was found on the example page.
